DP-25755: Fix behat test failure

arthurbaghdas opened the pull request:
**Description:**
Explain the technical implementation of the work done.


**Jira:** (Skip unless you are MA staff)
DP-25755


**To Test:**
- [ ] Add steps to test this feature


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

arthurbaghdas committed:
DP-25755: Fix behat test failure

arthurbaghdas committed:
Changelog

arthurbaghdas committed:
Entity embed array to string conversion issue fix

weitzman commented:
1. composer.lock is changing but I think thats a mistake?
2. Please add code comments about why we are doing unset() and when we can remove it.

arthurbaghdas committed:
Revert composer changes

arthurbaghdas committed:
Add comments

arthurbaghdas commented:
> 1. composer.lock is changing but I think thats a mistake?
> 
>     2. Please add code comments about why we are doing unset() and when we can remove it.

Reverted composer.lock changes, added comments.
